packages/fiori-freestyle-writer/src/packageConfig.ts

```diff
@@ -56,9 +59,9 @@ export function getPackageJsonTasks({
     return Object.assign(
         {
             start: startCommand,
-            'start-local': startLocalCommand,
-            'start-noflp': startNoFlpCommand
+            'start-local': startLocalCommand
```

Comment:
same comment here as before

Reply:
Please review again, I made the requested changes, thanks